packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -195,12 +205,27 @@ export interface ISemanticColors extends ISemanticTextColors {
    */
   inputPlaceholderBackgroundChecked: string;
 
+  /**
+   * The foreground of a control in rest state; e.g. unchecked toggle at rest color.
```

Comment:
"of an input"

Reply:
The example isn't really clear either. What's a foreground? The borders, labels, checkmark/dot are all foreground elements.

Reply:
@phkuo shouldn't all this description be given by a designer who approves an addition of a semanticColor slot? I would also expect that when we add a new slot we at least have 2 cases were it's a common color used and it proves the necessity of adding a new slot.

Reply:
In theory yes, design would provide the description. But in theory, a designer would be reviewing this PR instead of me =)

No, we need semanticSlots even if we only use them once. For instance, primaryButtonBackground. We only have one primary button, and AFAIK we don't use that slot elsewhere. But the point is that, if someone wanted to recreate that component, say in Silverlight, they'd be able to take our theme object and apply the appropriate semantic slots for it so that the design matched.

Reply:
There's the case for avoiding bloating the semantic slots offering. I know we've been discussing sometime in the future re-doing the colors offering altogether but in the meantime, while we have these semantic slots to work with, we should be careful to not add too many slots that aren't generalized/work for multiple cases.